package.json

```
@@ -23,7 +23,8 @@
     "lint:contracts": "solhint \"contracts/**/*.sol\" \"apps/*/contracts/**/*.sol\"",
     "lint:contract-tests": "eslint apps/*/test",
     "lint:wrapper": "eslint lib/",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "setup": "sh scripts/git-hooks/install"
```

```suggestion
    "setup-git-hooks": "sh scripts/git-hooks/install"
```

I find this name a bit confusing as it sounds it has something todo with the setup to install/run the code. We have `bootstrap`, `reset`, `install-all` - so I'd by a bit more specific here as so far this setup task is only installing the git hooks.

---

This setup is meant to include anything that is necessary, other than `npm install` to set up the repo itself.

---

Moved the entire thing to `postinstall` now.